src/renderer/components/PoolShares/PoolShares.styles.ts

```diff
@@ -78,11 +78,7 @@ export const InfoDescription = styled.div`
   margin-bottom: 20px;
 `
 
-export const AssetData = styled(AssetDataUI)`
-  & .ticker {
-    font-size: 16px;
-  }
-`
+export const AssetData = styled(AssetDataUI)``
```

Comment:
If we don't have any styles for `AssetData`, it can be used directly in `PoolShares` component